### DISCUSSION TITLE
I can't import modules whene virtual environment

### DISCUSSION THREAD

**Discussion by Diae98** [flet-dev/flet#1683] [Q&A]
i cant import flet whene create a virtual environment,
i did: 
python.exe -m venv .venv
.venv\Scripts\activate.bat
pip install flet --upgrade
flet create my-app
cd my-app

**Comment by FeodorFitsner** [comment]
What errors do you get?

**Reply by Diae98** [reply]
![Capture](https://github.com/flet-dev/flet/assets/108680745/10b42589-46ea-491f-9a89-9c1e13f53896)
you see..
